fe/fe-core/src/main/antlr4/org/apache/doris/nereids/DorisParser.g4

```diff
@@ -228,6 +228,7 @@ supportedShowStatement
     | SHOW DYNAMIC PARTITION TABLES ((FROM | IN) database=multipartIdentifier)?     #showDynamicPartition
     | SHOW EVENTS ((FROM | IN) database=multipartIdentifier)? wildWhere?            #showEvents
     | SHOW LAST INSERT                                                              #showLastInsert 
+    | SHOW ((CHAR SET) | CHARSET) wildWhere?                                        #showCharset
```

Comment:
```suggestion
    | SHOW ((CHAR SET) | CHARSET)                                      #showCharset
```
Since wildWhere is useless, we can remove it for simplicity

Reply:
done - pls review